addons/web/static/src/core/utils/draggable_hook_builder.js

```diff
@@ -662,8 +662,12 @@ export function makeDraggableHook(hookParams) {
                 // https://bugzilla.mozilla.org/show_bug.cgi?id=1352061
                 // https://bugzilla.mozilla.org/show_bug.cgi?id=339293
                 safePrevent(ev);
-                if (document.activeElement && !document.activeElement.contains(ev.target)) {
-                    document.activeElement.blur();
+                let activeElement = document.activeElement;
+                if (activeElement && activeElement.nodeName && activeElement.nodeName === "IFRAME") {
+                    activeElement = activeElement.contentDocument.activeElement;
+                }
```

Comment:
Your diff fixes the issue on a single level only (unlinkely to have nested iframes, but this is generic code)
```suggestion
                while (activeElement?.nodeName === "IFRAME") {
                    activeElement = activeElement.contentDocument.activeElement;
                }
```

Reply:
Thanks for the review, I fixed it